src/kongState.js

```diff
@@ -34,9 +34,9 @@ export default async (adminApi) => {
         const services = await adminApi.fetchServices();
         servicesWithPluginsAndRoutes = await Promise.all(services.map(async item => {
             const plugins = await adminApi.fetchServicePlugins(item.id);
-            const routes = await adminApi.fetchServiceRoutes(item.name);
+            const routes = await adminApi.fetchServiceRoutes(item.id);
```

Comment:
This was an unintentional use of the service name instead of id. It worked fine because kong accepts either as a primary id. Changing this anyways, since it was a red herring that tripped me up during some debugging.